src/main/resources/templates/db_scripts/schema.json

```
@@ -679,6 +679,12 @@
           "fieldName": "fullCallNumberNormalized",
           "sqlExpression" : "normalize_holdings_full_call_number(jsonb)",
           "sqlExpressionQuery": "normalize_call_number_string($) || '*'"
+        },
+        {
+          "fieldName": "electronicAccess",
```

Does it make sense to add `Uri` at the end, because the object is quite complex, and there are some other options that potentially can be used for search? 
```suggestion
          "fieldName": "electronicAccessUri",
```